var/spack/repos/builtin/packages/arborx/package.py

```diff
@@ -38,7 +38,8 @@ class Arborx(CMakePackage, CudaPackage, ROCmPackage):
         deprecated=True,
     )
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
```

Comment:
```suggestion
```
ArborX does not depend on C. Not sure what I am missing.

Reply:
@aprokop See https://github.com/spack/spack/pull/47662#issuecomment-2484192915

Reply:
ArborX declares CXX language in `project`. I don't see C compiler probed locally. Maybe it depends on dependencies and backend.

Reply:
I'll double check arborx